students/zelichowski_michal/lesson_01_python_basics/apple_sharing.py

```diff
@@ -0,0 +1,9 @@
+# N students take K apples and distribute them among each other evenly. The remaining (the undivisible) part remains in the basket.
+# How many apples will each single student get? How many apples will remain in the basket?
+#
+# The program reads the numbers N and K. It should print the two answers for the questions above.
```

Comment:
Możesz używać multiline commentów, mniej się namachasz z shiftem:)
Zerknij sobie ten wątek na SO: https://stackoverflow.com/questions/7696924/way-to-create-multiline-comments-in-python